js/web/lib/wasm/wasm-core-impl.ts

```diff
@@ -243,6 +247,24 @@ export const createSession = async(
   const outputNamesUTF8Encoded = [];
 
   try {
+
+    const contextIds: string[] = [];
```

Comment:
it seems why `contextIds` is an array is because the consideration of there are possible multiple entries in the execution provider array may contain webnn configurations. For example:
```
[
   { name: 'webnn', powerPreference: 'high-performance' },
   { name: 'webnn', powerPreference: 'default' },
}
```

I don't know if you want to deal with this situation. I think it may be enough to just look at the first configuration so that no need to use an array here.

Reply:
No longer applicable